pkg/csidriver/nodeserver.go

```diff
@@ -193,6 +194,15 @@ func (ns *nodeServer) NodePublishVolume(ctx context.Context, req *csi.NodePublis
 		return nil, err
 	}
 
+	s := csidriver.NewWebhook()
```

Comment:
We should not create a new webhook here, You should move the `ListedShareResource` function to another package (maybe a util package or something) and share it between this file and the one for the csidriver webhook.

Reply:
Make sense 👍